docs/EN_US/ECLStandardLibraryReference/SLR-Mods/CommonPrefix.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="CommonPrefix">
+  <title>CommonPrefix</title>
+
+  <para><emphasis role="bold">STD.Str.CommonPrefix<indexterm>
+      <primary>STD.Str.CommonPrefix</primary>
+    </indexterm><indexterm>
+      <primary>Str.CommonPrefix</primary>
+    </indexterm><indexterm>
+      <primary>CommonPrefix</primary>
+    </indexterm>(</emphasis> <emphasis>s1, s2 </emphasis><emphasis
+  role="bold">[ </emphasis><emphasis>,nocase</emphasis><emphasis role="bold">
+  ] )</emphasis></para>
+
+  <para><emphasis role="bold">STD.Uni.CommonPrefix<indexterm>
+      <primary>STD.Uni.CommonPrefix</primary>
+    </indexterm><indexterm>
+      <primary>Uni.CommonPrefix</primary>
+    </indexterm><indexterm/>(</emphasis> <emphasis>s1, s2 </emphasis><emphasis
+  role="bold">[ </emphasis><emphasis>,nocase</emphasis><emphasis role="bold">
+  ] )</emphasis></para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec colwidth="85*"/>
+
+      <colspec colwidth="915*"/>
+
+      <tbody>
+        <row>
+          <entry><emphasis>s1</emphasis></entry>
+
+          <entry>A string to compare.</entry>
+        </row>
+
+        <row>
+          <entry><emphasis>s2</emphasis></entry>
+
+          <entry>A string to compare.</entry>
+        </row>
+
+        <row>
+          <entry><emphasis>nocase</emphasis></entry>
+
+          <entry>Optional. If TRUE, the comparison is case-insensitive.If
+          omitted, the default is FALSE.</entry>
+        </row>
+
+        <row>
+          <entry>Return:</entry>
+
+          <entry>CommonPrefix returns either a STRING or UNICODE value, as
+          appropriate, containing the longest prefix common to both strings,
+          as copied from the first argument. The result is empty if the
```

Comment:
awkward sentence maybe better stated as: 
CommonPrefix returns either a STRING or UNICODE value, as appropriate. It contains the longest prefix common to both strings, as copied from the first argument.